app/src/main/java/com/example/movopfy/features/movies/presentation/ui/MoviesList.kt

```diff
@@ -18,14 +18,13 @@ import androidx.compose.ui.unit.dp
 import androidx.navigation.NavController
 import coil.compose.AsyncImage
 import com.example.movopfy.common.constants.API_CATEGORY_KINOPOISK
-import com.example.movopfy.network.kinopoisk.models.KinopoiskDocs
 import com.example.movopfy.uiComponents.navigation.Screen
 import com.example.movopfy.uiComponents.theme.dimensions
 
 @Composable
 fun MoviesList(
     modifier: Modifier = Modifier,
-    list: List<KinopoiskDocs>,
+    list: List<com.example.movopfy.features.movies.domain.models.KinopoiskItems>,
```

Comment:
please rename list and make normal import

Reply:
did